package.json

```diff
@@ -65,7 +65,10 @@
     "transform": {
       "^.+\\.json5?$": "json5-jest",
```

Comment:
If you just want to target `json5` here, you can remove the question mark `?`.  By doing so, you wouldn't need to specify the `transformIgnorePatterns` below.

Reply:
Made that change, thank you!